shaders/hdr/hlsl/gbuffer.frag.hlsl

```diff
@@ -97,7 +97,8 @@ FSOutput main(VSOutput input)
                 fresnel *= (1.0 - F0);
                 fresnel += F0;
 
-                float spec = (fresnel * geoAtt) / (NdotV * NdotL * 3.14);
+				// Note: clamp to zero to mitigate any divide by zero
```

Comment:
This clamping is questionable.
But as it's also done in the glsl shader, it could be addressed by the original author in a separate PR.

Reply:
It might be questionable, but without it our hardware produces the wrong answer.